src/routes/home/components/announcementDialog/data/announcementData.js

```diff
@@ -1,13 +1,20 @@
+// import { Card } from '../../../../../../tests/e2e/fixtures/cards';
+
 export const announcementData = {
-  id: 'opponentHandRevealed',
+  id: 'spades2024Announcement',
   activatorText: 'announcement.activatorText',
   title: 'announcement.title',
-  imgSrc: './img/announcement/deck-exhausted.gif',
   displayCards: [],
+  imgSrc: '/img/announcement/spades-2024-bracket.svg',
+  startTime: '2025-01-24',
+  endTime: '2025-01-26',
   announcementText: [
     {
       heading: 'announcement.heading',
-      paragraph: 'announcement.paragraph',
+      paragraph: [ 
+        { text: 'announcement.paragraph' },
+        { text: 'announcement.twitchLink', url: 'https://twitch.tv/cuttle_cards' }
+      ],
```

Comment:
The idea of switching from a string to an array is to allow embedding links in text while still being generic across translations. This way it will only render a link for objects with a url

Reply:
Chunks can also have a `heading` to add a heading above the paragraph. See BaseParagraph for the implementation